packages/core-components/src/lib/organisms/layout/sidebar/Sidebar.svelte

```diff
@@ -273,4 +274,14 @@
 			{/each}
 		</div>
 	{/if}
+	{#if builtWithEvidence}
+		<div class="fixed bottom-0 text-xs py-2">
+			<a
+				href="https://www.evidence.dev"
+				class="bg-gradient-to-r inline-block text-gray-950 antialiased font-medium"
+			>
+				Built with Evidence</a
```

Comment:
I think there's an argument to put this at the bottom of the article, rather than tucked away in the sidebar

Reply:
See gif -- I like that it's visible at all times, though think bottom right might be better?  

![CleanShot 2024-02-25 at 21 07 29](https://github.com/evidence-dev/evidence/assets/6857673/9fedc14b-175d-4387-b34f-23cd308d1338)